benefits/eligibility/views.py

```diff
@@ -9,11 +9,13 @@
 from django.utils.decorators import decorator_from_middleware
 from django.utils.translation import pgettext, gettext as _
 
+from eligibility_api import api
```

Comment:
Related to cal-itp/eligibility-api#26, I feel like we should be importing `client` (or even `Client` directly) here, not `api`, since the whole thing is `eligibility_api`.

Since the API spec is split into a client/server model, it makes sense to organize the code around that too.